app/jobs/document_proofing_job.rb

```diff
@@ -58,6 +60,7 @@ def perform(
           selfie_image: selfie_image || '',
           image_source: image_source(image_metadata),
           liveness_checking_enabled: liveness_checking_enabled,
+          applicant: applicant,
```

Comment:
`applicant` is really generic... and makes it hard to track down which attributes are actually being used (since almost anything could be inside this hash), WDYT of just passing `uuid` and `uuid_prefix` keys?

```suggestion
          user_uuid: applicant[:uuid],
          uuid_prefix: applicant[:uuid_prefix],
```

Reply:
Sounds good - making the changes - they percolate through...